doc/glossary.rst

```diff
@@ -0,0 +1,85 @@
+Glossary
+========
+
+A glossary of terms used in the module, with a description of how they are used, as well as common abbreviations.
+Note, these terms are defined using peak-centered cycles.
+
+Shape
+-----
+Features that describe the shape of a cycle.
+
+.. glossary::
+
+    Period (``period``)
+        A single cycle of a rhythm, defined as the time between two consecutive troughs (or peaks).
+
+    Peak (``time_peak``)
```

Comment:
this feels a slightly odd definition for peak / trough, and I think _not_ how they are computed. 
The peak / trough values are computed as the max / min values of detected cycles (where cycles are operationally defined as 'time between zero-crossings'. 

I think 'time between' doesn't quite capture this, and is slightly different from the intuitive idea of 'max' / 'min'.

So, maybe something like:
```
Peak
    The point of maximum height of a cycle. 
```

Does this track?

Reply:
I think the definition for `Peak (time_peak)` is correct, it's the time between adjacent zero-crossings. It is defined in the docstring as the `time between rise and decay zero-crossing`, and confirmed with:
```
>>> start = df_samples.iloc[0]['sample_zerox_rise']
>>> end = df_samples.iloc[0]['sample_zerox_decay']
>>> len(sig[start:end]*fs) == df_features.iloc[0]['time_peak']
True
```
The definition you gave, I think, is for this (which is a different entry in the glossary):
```
Voltage Peak (volt_peak)
        The voltage at the current peak.
```

I can see the confusion here though. `time_peak` sounds like it should refer to the time at which the peak occurs, which is instead stored in `df_samples` as `sample_peak` (and must be multiplied by fs to give the actual time the peak occurs). Maybe it `time_peak` should be redefined as `duration_peak`?

Reply:
Also, I should probably also include a glossary section for the columns defined in `df_samples`.

Reply:
Ohh, I see. Yeh, I had mis-interpreted this term (as something equivalent to "sample-peak"). 

I don't think the name is totally ideal. But this PR is _docs_ so, let's not start poking at names here - let's keep this name in mind, and do a sweep of API & names on the release candidate.